UPGRADE.md

```diff
@@ -755,9 +755,17 @@ test('MyComponent', () => {
 });
 ```
 
+## AutocompleteInput Now Uses Material UI Autocomplete
+
+We migrated the `AutocompleteInput` so that it leverage Material UI [`<Autocomplete>`](https://mui.com/components/autocomplete/). If you relied on [Downshift](https://www.downshift-js.com/) options, you'll have to update your component.
```

Comment:
```suggestion
We migrated the `AutocompleteInput` so that it leverages Material UI [`<Autocomplete>`](https://mui.com/components/autocomplete/). If you relied on [Downshift](https://www.downshift-js.com/) options, you'll have to update your component.
```